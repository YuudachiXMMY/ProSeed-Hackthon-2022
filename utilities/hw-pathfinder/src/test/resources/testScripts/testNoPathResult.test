# Test Dijkstra's Algorithm with no path in a graph

CreateGraph graph

AddNode graph A
AddNode graph B

FindPath graph A B