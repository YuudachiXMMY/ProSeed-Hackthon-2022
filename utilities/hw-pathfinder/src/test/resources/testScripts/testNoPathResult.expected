# Test Dijkstra's Algorithm with no path in a graph

created graph graph

added node A to graph
added node B to graph

path from A to B:
no path found